quisby.py

```diff
@@ -459,12 +459,20 @@ def compare_data(s_list, comp_list, noti_flag, exclude):
     noti_flag = True
     if args.no_notify:
         noti_flag = False
+    
+    default_config_path = os.path.expanduser("~/.quisby/config/config.ini")
 
-    if not args.config:
-        custom_logger.error("No configuration path mentioned.")
-        exit(1)
-    else:
+    if args.config:
         util.config_location = args.config
+    else:
+        if os.path.exists(default_config_path):
+            custom_logger.warning(f"No --config argument provided. Using default: {default_config_path}")
+            util.config_location = default_config_path
+        else:
+            custom_logger.error(f"No configuration provided and default config not found at: {default_config_path}")
+            exit(1)
```

Comment:
We shouldn't check for config existence in 2 different locations.

If this was missing before, then just have it check for `util.config_location` and if it doesn't exist, exit out.